CMakeLists.txt

```diff
@@ -20,7 +20,13 @@ set(EXTENSION_DEPENDS SlicerIGSIO)
 # Extension dependencies
 find_package(Slicer REQUIRED)
 include(${Slicer_USE_FILE})
-find_package(SlicerIGSIO REQUIRED)
+
+if (NOT DEFINED Slicer_EXTENSION_SOURCE_DIRS)
+  find_package(SlicerIGSIO REQUIRED) 
+else()
+  # Allow usage in custom applications

```

Comment:
I'm not sure if this comment is accurate. The difference should not be due to bundling the extension into Slicer but because there was a reorganization of libraries between packages in recent Slicer versions.

Reply:
I can change this to
`# Allow usage for bundled extensions`
I wasn't sure what were bundled extensions, but based on what I found in the code, such as 
https://github.com/Slicer/Slicer/blob/96ee24860f7e9cb7364f5e72872e6d0a0cbd097f/CMake/SlicerExtensionCPackBundleFixup.cmake.in#L230-L234
it seems that it is primarily for custom apps. Up to you, I don't mind either way, just wanted to document a bit why the difference.

Reply:
Maybe better I guess
`# Allow usage if dependent extension is bundled`